Build failure on iOS

I am using your plugin/library in commonTest to read in some resource file text. This works nicely for desktop and Android but fails at build time for iOS. The message is:
```
FAILURE: Build failed with an exception.
* What went wrong:
A problem was found with the configuration of task ':composeApp:assembleIosX64TestResources' (type 'AssembleTargetResourcesTask').
  - Gradle detected a problem with the following location: '/Users/mpaus/Projects/tmp/KotlinWasmBridgeDemo/composeApp/build/generated/compose/resourceGenerator/assembledResources/iosX64Test'.
    
    Reason: Task ':composeApp:copyResourcesDebugTestIosX64' uses this output of task ':composeApp:assembleIosX64TestResources' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.
    
    Possible solutions:
      1. Declare task ':composeApp:assembleIosX64TestResources' as an input of ':composeApp:copyResourcesDebugTestIosX64'.
      2. Declare an explicit dependency on ':composeApp:assembleIosX64TestResources' from ':composeApp:copyResourcesDebugTestIosX64' using Task#dependsOn.
      3. Declare an explicit dependency on ':composeApp:assembleIosX64TestResources' from ':composeApp:copyResourcesDebugTestIosX64' using Task#mustRunAfter.
    
    For more information, please refer to https://docs.gradle.org/8.11.1/userguide/validation_problems.html#implicit_dependency in the Gradle documentation.
```
I guess this should work without any extra steps on my side for iOS.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,7 +11,7 @@ Notable changes are documented in this file, whose format follows [Keep a Change
 ### Fixed
 
 - Handle all native test binaries instead of just the first
-- Compose Multiplatform compatibility on Apple platforms (#141)
+- Compose Multiplatform compatibility on iOS (#141)
 
 ## [0.11.0] - 2025-12-13
 

--- a/resources-plugin/src/main/kotlin/ResourcesPlugin.kt
+++ b/resources-plugin/src/main/kotlin/ResourcesPlugin.kt
@@ -72,15 +72,15 @@ class ResourcesPlugin : KotlinCompilerPluginSupportPlugin {
                     dependantTasks = listOf(binary.linkTaskName)
                 )
 
-                if (isAppleCompilation(kotlinCompilation)) {
-                    // HACK: Avoid task dependency conflicts with Compose Multiplatform on Apple platforms.
+                if (isIosCompilation(kotlinCompilation)) {
+                    // HACK: Avoid task dependency conflicts with Compose Multiplatform on iOS.
                     val composeResourceTasks = project.tasks.matching { task ->
                         task.name.startsWith("assemble") &&
                             task.name.contains(target.targetName) &&
                             task.name.endsWith("TestResources")
                     }
                     copyResourcesTask.configure { it.mustRunAfter(composeResourceTasks) }
-                } else {
+                } else if (!isAppleCompilation(kotlinCompilation)) {
                     project.tasks.withType(KotlinNativeTest::class.java) { testTask ->
                         testTask.workingDir = binary.outputDirectory.absolutePath
                         testTask.dependsOn(copyResourcesTask)
@@ -134,6 +134,10 @@ class ResourcesPlugin : KotlinCompilerPluginSupportPlugin {
 
     private fun isAppleCompilation(kotlinCompilation: KotlinNativeCompilation): Boolean {
         return kotlinCompilation.konanTarget.family.isAppleFamily
+    }
+
+    private fun isIosCompilation(kotlinCompilation: KotlinNativeCompilation): Boolean {
+        return kotlinCompilation.konanTarget.family == Family.IOS
     }
 
     private fun isJsNodeCompilation(kotlinCompilation: KotlinCompilation<*>): Boolean {

